packages/solana/src/utils/chains.ts

```diff
@@ -0,0 +1,23 @@
+export const SOLANA_CHAINS = [
+  "solana:mainnet",
+  "solana:devnet",
+  "solana:testnet",
+  "solana:localnet",
+]
```

Comment:
If we add `as const` here then `typeof SOLANA_CHAINS` becomes `"solana:mainnet" | "solana:devnet" | etc` instead of `string`. If that's what we want!

```suggestion
] as const
```